edgecompute/examples/traffic-routing/url-shortner/edgekv.js

```diff
@@ -0,0 +1,328 @@
+/*

```

Comment:
We should never include "edgekv.js" in any of our public examples. Instead, the documentation should instruct the user to download the latest from https://github.com/akamai/edgeworkers-examples/tree/master/edgekv/lib